### PR TITLE
Revert "Revert "Update dot env""

### PR DESCRIPTION
**Pull request by adamrmelnyk**
Reverts amaabca/capistrano-amadeploy#15

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:hankey: 
Why was it reverted, and what changed to make it ok to reimplement these changes?

**Comment by adamrmelnyk** [comment]
Before this one goes in, we essentially need to do a bit more and a change a few things with the unicorns. Ideally we would just move off of this completely and just update capistrano though that will take some time as well.

**Comment by adamrmelnyk** [comment]
:wave: The unicorns have been changed.

File: spec/multistage_spec.rb [hunk 1]
```diff
@@ -16,32 +16,32 @@
...
+    expect(config.current_stage).to eq 'development'
+    expect(config.foo).to eq 'bar'
   end
 
   it 'aborts when no stage selected' do
     with_stderr do |output|
       config.unset :default_stage
       expect { cli_execute 'example' }.to raise_error(SystemExit)
-      output.should include('No stage specified. Please specify one of: development, production')
+      expect(output.include? 'No stage specified. Please specify one of: development, production').to be true
```
Comment (mvandenbeuken) [inline spec/multistage_spec.rb]: can this be ```expect(output).to include 'No stage specified. Please specify one of: development, production'```?

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
There are a bunch of other `expect(x.include? y).to eq true` in there, if you're making that change.

**Comment by adamrmelnyk** [comment]
Alright I shall do that then

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by adamrmelnyk** [comment]
Tested deploying with my_ama. seems to be working properly

**Comment by mvandenbeuken** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**